mp/src/game/client/neo/c_neo_player.cpp

```diff
@@ -594,6 +594,12 @@ extern ConVar mat_neo_toc_test;
 int C_NEO_Player::DrawModel(int flags)
 {
 	int ret = 0;
+
+	if (flags & STUDIO_IGNORE_NEO_EFFECTS || !(flags & STUDIO_RENDER))
```

Comment:
Nit: Could we move this `ret` declaration below the early BaseClass return just below here, so after line 602? Just so it's clear we don't actually use it as the retval for that case.